### ISSUE TITLE
Solution of the kingdom for using pinfo

### ISSUE THREAD

**Issue by Desarrollo-zeros** [Arkania/ArkCORE-NG#195]
-- Remove entry == 548 --> Player%s %s (guid: %u) Account: %s (id: %u) Email: %s GMLevel: %u Last IP: %s Last login: %s Latency: %ums
++ Add entry == 548 --->Ã¢â€â€š GM Mode active, Phase: -1 
 Arkore_string Pinfo crash crashes server
![image](https://cloud.githubusercontent.com/assets/23331674/20400355/61e5334c-acc2-11e6-9554-6897f3de9bf4.png)

**Comment by gpn39f** [comment]
the crypto on each line.. Ã¢â€�â€š point on  corrupted spanish langue 
if you can help to repair damaged spanish files: write me PM in our forum
http://gpn39f.spdns.de/forum/

**Comment by Desarrollo-zeros** [comment]
ok amigo veré para ayudarte

**Comment by Desarrollo-zeros** [comment]
yo uso en mi DB -->entry =  548  content_default =│ GM Mode active, Phase: -1

**Comment by gpn39f** [comment]
yes.. the crypt should be an | on the left side of .pinfo output.
we use utf8 general ci for our table,
The cryptic characters are an indication that something other than utf8 has destroyed the table.

**Comment by gpn39f** [comment]
have you fixed your problem???
